system-variables.md

```diff
@@ -2319,6 +2319,12 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 
 This variable is an alias for `transaction_isolation`.
 
+### tx_isolation_one_shot
+
+This variable is an internal system variable, not meant to be used directly.
```

Comment:
Could you provide the scope, the default value, or type, and the description?

Reply:
People should not use this variable directly, so why bother for more details?

Reply:
(For some historial reasons) It's a special one, not like the other variables...